base_marker_id: Channel705
head_marker_id: Channel706
freq: 50
kp_x: 0.06
kp_y: 0.06
kp_z: 0.0
kp_roll: 0.0
kp_pitch: 0.0
kp_yaw: 0.0
kd_x: 0.005
kd_y: 0.005
kd_z: 0.0
kd_roll: 0.0
kd_pitch: 0.0
kd_yaw: 0.0
ki_x: 0.0
ki_y: 0.0
ki_z: 0.0
ki_roll: 0.0
ki_pitch: 0.0
ki_yaw: 0.0
